_layouts/default.html

```diff
@@ -63,6 +63,60 @@
         <script defer src="/js/bootstrap.min.js"></script>
         <script defer src="/js/scripts.js"></script>
 
+        <!-- Schema.org JSON-LD Markup for Search Engine Optimization -->
+        <script type='application/ld+json'>
+        {
+            "@context": "https://schema.org/",
+            "@type": "SoftwareApplication",
+            "applicationCategory": "FinanceApplication",
+            "name": "Bisq",
+            "image": "https://bisq.network/images/bisq-logo.svg",
+            "availableOnDevice": [
+                "Desktop",
+                "Smartphone",
+                "Tablet"
+            ],
+            "author": {
+            "@type": "Organization",
+            "url": "https://bisq.network/dao/",
+            "name": "Bisq Decentralized Autonomous Organization",
+            "sameAs": [
+                "https://github.com/bisq-network",
+                "https://twitter.com/bisq_network",
+                "https://www.youtube.com/c/bisq-network",
+                "https://bisq.network/slack-invite",
+                "https://bisq.community/",
+                "https://lists.bisq.network/listinfo/bisq-contrib",
+                "https://webchat.freenode.net/?channels=bisq",
+                "https://www.reddit.com/r/bisq",
+                "https://keybase.io/team/bisq"
+	          ]
+            },
+            "downloadUrl": "https://bisq.network/downloads/",
+            "fileSize": "180MB",
+            "operatingSystem": [
+                "Windows",
+                "Linux",
+                "OSX",
+                "iOs",
+                "Android"
+            ],
+            "releaseNotes": "https://github.com/bisq-network/bisq/releases",
+            "screenshot": "https://bisq.network/images/bisq_screen0.svg",
+            "softwareVersion": "{{ site.client_version }}",
+            "aggregateRating": {
+            "@type": "AggregateRating",
+            "ratingValue": "5",
+            "ratingCount": "20"
+          },
+          "offers": {
```

Comment:
Bisq itself doesn't exchange bitcoin, so I think it's better to omit this too.

Reply:
> Bisq itself doesn't exchange bitcoin, so I think it's better to omit this too.

`offers' means an offer to sell the app.

If the app is free of charge, we should set offers.price to 0. For example:
```
"offers": {
  "@type": "Offer",
  "price": "0"
}
```

If the app has a price greater than 0, you must include offers.currency. For example:

```
"offers": {
  "@type": "Offer",
  "price": "1.00",
  "priceCurrency": "USD"
}
```

In this particular case, I'd prefer to set 0 as price and set BTC as priceCurrency